src/components/relationships-panel/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/relationships-panel",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

Comment:
Don't change this -- Lerna does this.

Reply:
PS steps 14 and 15 can only be done by a repo admin (since they affect the main branch directly).

Reply:
Whoops, thanks for the reminder.  I'll fix it.